### DISCUSSION TITLE
Getting a grip on forest science

### DISCUSSION THREAD

**Discussion by jkshuman** [NGEET/fates#898] [Papers]
Recent Nature issue spotlights work on the importance of forests globally. The lead editorial linked here highlights supporting the scientists studying forests globally, and the need for more interdisciplinarity work, such as that happening with FATES. 

https://www.nature.com/articles/d41586-022-02182-0

[Editorial_2022_Nature_Importance_forest_science.pdf](https://github.com/NGEET/fates/files/9500672/Editorial_2022_Nature_Importance_forest_science.pdf)

**Comment by rosiealice** [comment]
Bbbnñpm

-------------------------
Dr Rosie A. Fisher

Senior Researcher
CICERO Center for International Climate Research
Oslo, Norway
https://cicero.oslo.no<https://cicero.oslo.no/>
________________________________
From: jkshuman ***@***.***>
Sent: Tuesday, September 6, 2022 11:00:27 PM
To: NGEET/fates ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [NGEET/fates] Getting a grip on forest science (Discussion #898)


Recent Nature issue spotlights work on the importance of forests globally. The lead editorial linked here highlights supporting the scientists studying forests globally, and the need for more interdisciplinarity work, such as that happening with FATES.

https://www.nature.com/articles/d41586-022-02182-0

Editorial_2022_Nature_Importance_forest_science.pdf<https://github.com/NGEET/fates/files/9500672/Editorial_2022_Nature_Importance_forest_science.pdf>
